### ISSUE TITLE
Null check operator used on a null value

### ISSUE THREAD

**Issue by kuhnroyal** [marcglasberg/async_redux#106]
I am using `WaitAction.remove` and it seems the key/ref doesn't exist.
That the key does not exist is of course my fault, but there should ne be a force cast in the `Wait`.

```dart
Set<Object?> refs = newFlags[flag]!;
```

**Comment by marcglasberg** [comment]
This bang was added automatically by the migrate tool.

What do you think I should be doing here? Just ignore it when the key does not exist? 

```
Set<Object?> refs = newFlags[flag] ?? {};
```

Throw an error? 

```
Set<Object?> refs = newFlags[flag] ?? {};
```

I guess first option, right?

**Comment by kuhnroyal** [comment]
Yea, I would opt for not doing anything. This is how it was until NNBD.

**Comment by marcglasberg** [comment]
Fixed in version [11.0.0].
